shop/src/main/java/ro/msg/learning/shop/repository/StockRepository.java

```diff
@@ -0,0 +1,10 @@
+package ro.msg.learning.shop.repository;
+
+import ro.msg.learning.shop.entity.Stock;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+@Repository
+public interface StockRepository extends JpaRepository<Stock, UUID> {
```

Comment:
Instead of UUID it should be StockID as StockID is the type of the id in the StockEntity